850.split-ambiguities/d.yaml

```diff
@@ -126,6 +126,10 @@
 
 - { name: disktype, wwwpart: kamwoods, setname: disktype-kamwoods } # fork
 
+- { name: distribution, wwwpart: [philovivero, time-less-ness, wizzat], setname: "python:distribution" }
+- { name: distribution, wwwpart: [docker, github.com/distribution/], setname: distribution-cncf }
```

Comment:
I'd pick a different target name, `cncf` is not descriptive. May reuse `docker-distribution`, or pick `distribution-docker` for consistency.

Reply:
Officially, it's no longer part of the Docker GitHub organisation and the "Docker" part of its name has been dropped: https://www.docker.com/blog/donating-docker-distribution-to-the-cncf/. It also not Docker-specific (e.g. GitHub Container Registry is supported).

Given it's an implementation for the OCI Distribution spec, would `distribution-oci` make sense?